src/main/java/vention/customWebElement/WebElementImp.java

```diff
@@ -31,15 +31,23 @@ private WebElement waitElementToBeVisible(WebElement element) {
   }
 
   private WebElement waitElementToBeClickable(WebElement element, int seconds) {
-    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds)).until(ExpectedConditions.elementToBeClickable(element));
+    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds))
+        .until(ExpectedConditions.elementToBeClickable(element));
   }
 
   private WebElement waitElementToBeVisible(WebElement element, int seconds) {
-    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds)).until(ExpectedConditions.visibilityOf(element));
+    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds))
+        .until(ExpectedConditions.visibilityOf(element));
   }
 
   public static void clickByLocator(By locator, int seconds) {
```

Comment:
1) This method should not be static, it breaks the logic of the class.
The class should represent webelement wrapper but finding elements is logic of page factory. WebElementImpl should have only the methods that can be used directly to an existing webelement

2) let's implement 'format()' method which allows to add locators with '%s' to annotations and formats them. I will send an example 

3) I'd recommend to rename WebElementImpl to smth like 'CustomWebElement' or any similar. We need a name from business logic perspective